.gitignore

```diff
@@ -7,3 +7,34 @@ train.sh
 build/
 cmake-build-debug
 .idea/*
+*.o
+*.Po
+compile
+config.guess
+config.sub
+depcomp
+install-sh
+missing
```

Comment:
please remove these files from .gitignore

Reply:
everytime you build the porject using ./configure they will be added! so if you wan them in the repo please add them in repo if you do not want them in repo then you must add them to gitignore!